src/table.scss

```diff
@@ -418,6 +418,24 @@ $block: #{$fd-namespace}-table;
     justify-content: space-between;
   }
 
+  &__column-header-icons {
+    display: flex;
+
+    fd-icon:not(:last-child) {
```

Comment:
I believe you are targeting an Angular component `fd-icon`. This can't be done in Fund-styles. If the css is specific for the Angular implementation it should be in fund-ngx.

Reply:
fixed